src/@types/ButtonVariant.ts

```diff
@@ -0,0 +1,6 @@
+export enum ButtonVariant {
+  PRIMARY = "PRIMARY",
+  SECONDARY = "SECONDARY",
+  DANGER = "DANGER",
+  GHOST = "GHOST",
+}
```

Comment:
Hm.. I'm thorned between putting this in the button file, or in `/@types/components/ButtonVarian`.

Do you have a preference?

Reply:
Actually I was following the rule of moving a type to its own file if it's used in several files, for me its suitable like this, the only advantage of moving it back to the component file is that we could have a direct overview of the variant's list. Do you want me to fix it?

Reply:
> Actually I was following the rule of moving a type to its own file if it's used in several files

Yes, and I'm glad you did it :). It's just that `@/types` is gonna be massive if we put the components' types at the root. Let's do `@types/components/*` for now :).

Reply:
OK fixed 87d4ee7